dev_environment/setup_dev_env.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
Can this script be run in github CI/CD in a github workflow?

Reply:
Not at moment.  But if we update the CI/CD github workflow, it should be added in.